src/components/Extension/PrivateExtensions/actions/IntakeScheduling/IntakeScheduling.tsx

```diff
@@ -57,14 +57,24 @@ export const IntakeScheduling: FC<IntakeSchedulingProps> = ({
   }, [updateLayoutMode, resetLayoutMode])
 
   const fetchProviderFn = useCallback(
-    (providerId: string) =>
-      fetchProvider({ input: { providerId }, requestOptions: { baseUrl } }),
+    async (providerId: string) => {
+      const provider = fetchProvider({
```

Comment:
are you supposed to be awaiting this `fetchProvider` function as well?

Reply:
good point!